### PR TITLE
Update reference values for messages in messages.pot

### PR DESCRIPTION
**Pull request by antonsoroko**
## Description

Update reference values for messages in messages.pot
fixes https://github.com/elgatito/plugin.video.elementum/issues/725

after we agree about changes i will update `msgid` in all translations.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Translation

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: resources/language/messages.pot [hunk 1]
```diff
@@ -1814,15 +1814,15 @@ msgid "Torrents autoload on plugin startup"
...
-msgid "Connection limits"
+msgid "Limits for connections"
 msgstr ""
 
 msgctxt "#30592"
 msgid "Seeding"
 msgstr ""
 
 msgctxt "#30593"
-msgid "Seed forever (unless disk space needed)"
```
Comment (antonsoroko) [inline resources/language/messages.pot:1825]: кажется автоудаления сейчас нет, или я его как-то пропустил

**Comment by antonsoroko** [comment]
@elgatito any comments?

**Comment by elgatito** [comment]
@antonsoroko looks fine.

**Comment by antonsoroko** [comment]
@elgatito i updated all langs strings.po, so now it is ready to merge. thanks.

**Comment by antonsoroko** [comment]
@elgatito замержим?

**Comment by elgatito** [comment]
```
./scripts/xgettext.sh
```
Проходит проверку?

**Comment by antonsoroko** [comment]
@elgatito 

да
```
$ git branch --show-current
update-reference-values-in-messages.pot
$ scripts/xgettext.sh
Checking resources/language/Croatian/strings.po                        [  OK  ]
Checking resources/language/Dutch/strings.po                           [  OK  ]
Checking resources/language/English/strings.po                         [  OK  ]
Checking resources/language/Finnish/strings.po                         [  OK  ]
Checking resources/language/French/strings.po                          [  OK  ]
Checking resources/language/German/strings.po                          [  OK  ]
Checking resources/language/Greek/strings.po                           [  OK  ]
Checking resources/language/Hebrew/strings.po                          [  OK  ]
Checking resources/language/Hungarian/strings.po                       [  OK  ]
Checking resources/language/Italian/strings.po                         [  OK  ]
Checking resources/language/Portuguese (Brazil)/strings.po             [  OK  ]
Checking resources/language/Portuguese/strings.po                      [  OK  ]
Checking resources/language/Romanian/strings.po                        [  OK  ]
Checking resources/language/Russian/strings.po                         [  OK  ]
Checking resources/language/Slovak/strings.po                          [  OK  ]
Checking resources/language/Spanish (Argentina)/strings.po             [  OK  ]
Checking resources/language/Spanish (Mexico)/strings.po                [  OK  ]
Checking resources/language/Spanish/strings.po                         [  OK  ]
Checking resources/language/Swedish/strings.po                         [  OK  ]
Checking resources/language/Ukrainian/strings.po                       [  OK  ]
```

**Pull request merged**